src/LintingViewExtension/LintingViewExtension.csproj

```diff
@@ -0,0 +1,130 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(SolutionDir)/Config/CS.props" />
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{C86F9058-229D-40A9-95D5-D6F081AA9230}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Dynamo.LintingViewExtension</RootNamespace>
+    <AssemblyName>LintingViewExtension</AssemblyName>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+    <Deterministic>true</Deterministic>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>..\..\bin\AnyCPU\Debug\</OutputPath>
```

Comment:
I think these projects should probably be setup with the imported outputs and assembly shared info generator, unless you want this assembly to have a different version number purposefully.

Reply:
@mjkkirschner hm, should be setup with the Assembly Shared info, but not sure i know about the `imported output paths` where are they coming from?

Reply:
the output paths should be coming from:
https://github.com/DynamoDS/Dynamo/blob/master/src/Config/CS.props#L7
if you are importing this - I think if you define them again here you'll overwite that import if it appears further up in the csproj file.

Reply:
Yea i was importing it, not sure how i ended up overwriting it.. Should be fixed now.